src/fondue/Fondue.js

```diff
@@ -269,4 +269,47 @@ export default class Fondue {
 			return "";
 		}
 	}
+
+	// Returns an array of all supported Unicode characters
+	// from the "best" cmap.
+	get supportedCharacters() {
+		const cmap = this.getBestCmap();
+		const chars = [];
+		for (const chunk of cmap) {
```

Comment:
This will break if `this.getBestCmap()` returns `false`/`null`. Can you add some error handling here?